shotover-proxy/src/message/mod.rs

```diff
@@ -321,6 +325,7 @@ pub enum Value {
     None,
     #[serde(with = "my_bytes")]
     Bytes(Bytes),
+    Bigint(BigInt),
```

Comment:
is this different to Varint?

Reply:
ah... maybe this got left in from a bad merge?

Reply:
Wouldn't we want to know whether it was a BigInt or VarInt when writing back

Reply:
Are you referring to cassandra bigint? I believe we decided to represent that as Integer(_, IntSize::U64).
Cassandra Varint is shotover Varint.

Reply:
*IntSize::I64, but yes I remember that.